go.mod

```diff
@@ -75,7 +75,7 @@ require (
 	github.com/crewjam/saml v0.4.13
 	github.com/davecgh/go-spew v1.1.1 // indirect
 	github.com/docker/distribution v2.8.2+incompatible
-	github.com/docker/docker v23.0.2+incompatible
+	github.com/docker/docker v23.0.6+incompatible
```

Comment:
Curious why we need this change when I believe it's overridden by change in `replace` block (backed up by the fact there are no changes in `go.sum`)?
Now after typing the above, I see you state "removes false positives being flagged due to Docker's version being replaced.". Is this something that can be fixed/improved in that flagging process?

Reply:
I would concur, this change is redundant given that is not the version that Go is loading (due to the `replace` directive):

![image](https://github.com/rancher/rancher/assets/5452977/562e962b-e5ee-43ef-80d7-919afac4aecf)

Reply:
Yes, you are right. We are bumping `github.com/docker/docker` to remove the noise from some tools that can't reliably identify the `replace ( github.com/docker/docker => github.com/docker/docker )`. Our tools can reliably identify this, but Dependabot sometimes trigger them, as well as customers' reports.

Reply:
@macedogm , we're good to merge both bumps. I've also got OK from @kinarashah in team's discussion yesterday so we can count her as approver as well.